src/ocm/components/clusterConfiguration/staticIp/commonValidationSchemas.tsx

```diff
@@ -53,16 +57,36 @@ const isValidAddress = (protocolVersion: ProtocolVersion, addressStr: string) =>
   }
 };
 
-const isReservedAddress = (ip: string, protocolVersion: ProtocolVersion) => {
+const isReservedAddress = (ip: string, protocolVersion: ProtocolVersion, isDnsAddress = false) => {
   try {
+    console.log(protocolVersion);
+
     if (protocolVersion === ProtocolVersion.ipv4) {
-      return ip === RESERVED_LOCAL_HOST_IPS.ipv4 || ip === RESERVED_CATCH_ALL_IPS.ipv4;
+      if (isDnsAddress) {
+        return (
+          ip === RESERVED_DNS_LOCALHOST_IP ||
+          ip === RESERVED_DNS_BROADCAST_IP ||
+          ip === RESERVED_LOCAL_HOST_IPS.ipv4 ||
+          ip === RESERVED_CATCH_ALL_IPS.ipv4
+        );
+      } else {
+        return ip === RESERVED_LOCAL_HOST_IPS.ipv4 || ip === RESERVED_CATCH_ALL_IPS.ipv4;
```

Comment:
Not sure about this, but I don't think the gateway address can be the broadcast either, so do we need this "IF"?

Reply:
The bug if for Dns address. From gateway address is the same?

Reply:
No, you cannot use a broadcast/loopback address for gateway either. These are special addresses reserved for special use. Having a list of these reserved addresses is the way to go, imo.